Jenkinsfile

```diff
@@ -255,7 +270,7 @@ spec:
         }, failFast: true)
 
     def platforms = [:]
-
+/*
```

Comment:
Still need to restore testing on other platforms.  You may need to change their cleanup code to follow your `try/finally` refactor.

Reply:
I'm on it :-)